src/KDTree/KDTree.h

```diff
@@ -44,7 +44,11 @@
 #include <iostream>
 #include <vector>
 #include <utility>
-#include <algorithm> 
+#include <algorithm>
+
+//option list for the performance test
+#include "/home/jinsu/dev/VELOCIraptor-STF/src/js_perform.h"
```

Comment:
This include should be removed I assume. Please check that the code compiles without this include